galite-tests/src/test/kotlin/org/kopi/galite/tests/form/FormTests.kt

```diff
@@ -31,67 +33,68 @@ import org.kopi.galite.tests.JApplicationTestBase
 import org.kopi.galite.visual.WindowController
 
 class FormTests: JApplicationTestBase() {
-  object User: Table() {
-    val id = integer("id")
-    val name = varchar("name", 20)
-    val age = integer("age")
+
+  @Test
+  fun sourceFormTest() {
+    val formModel = TestForm.model
+    assertEquals(TestForm::class.qualifiedName!!.replace(".", "/"), formModel.source)
   }
+}
 
-  object TestForm: Form() {
-    override val locale = Locale.FRANCE
-    override val title = "form for test"
+object User: Table() {
+  val id = integer("id")
+  val name = varchar("name", 20)
+  val age = integer("age")
+}
 
-    val graph = actor (
-      menu =  "Action",
-      label = "Graphe",
-      help =  "Representer les valeurs en graphe"
-    ) {
-      key  =  KeyEvent.VK_F9  // key is optional here
-      icon =  "column_chart"  // icon is optional here
-    }
+object TestForm: Form() {
+  override val locale = Locale.FRANCE
+  override val title = "form for test"
 
-    init {
-      page("test page") {
-        val testBlock = block(1, 1, "Test", "Test block") {
-          val u = table(FormTests.User)
-          val i = index(message = "ID should be unique")
+  val graph = actor (
+          menu =  "Action",
+          label = "Graphe",
+          help =  "Representer les valeurs en graphe"
+  ) {
+    key  =  KeyEvent.VK_F9  // key is optional here
+    icon =  "column_chart"  // icon is optional here
+  }
 
-          val id = hidden(Domain<Int>(20)) {
-            label = "id"
-            help = "The user id"
-            columns(u.id)
-          }
-          val name = mustFill(Domain<String>(20)) {
-            label = "name"
-            help = "The user name"
-            columns(u.name)
-          }
-          val age = visit(Domain<Int>(3)) {
-            label = "age"
-            help = "The user age"
-            columns(u.age) {
-              index = i
-              priority = 1
-            }
+  init {
+    page("test page") {
+      val testBlock = block(1, 1, "Test", "Test block") {
+        val u = table(User)
+        val i = index(message = "ID should be unique")
+
+        val id = hidden(domain = Domain<Int>(20)) {
+          label = "id"
+          help = "The user id"
+          columns(u.id)
+        }
+        val name = mustFill(domain = Domain<String>(20), position = at(1, 1)) {
```

Comment:
Implement more test cases for the  "at" dsl  to go through the different functions